### ISSUE TITLE
Release channels for apps

### ISSUE THREAD

**Issue by mitar** [sandstorm-io/sandstorm#1927]
It would be great if apps would provide release channels. So that users could decide for example between stable and experimental channels.

**Comment by paulproteus** [comment]
I agree that this would be great. It's not something on our agenda for the next couple of months. I think the things needed here are:

- A technical design of how that would integrate into the app market's JSON output

- A technical design of how apps might interact with this, such as proposed `spk upload` commands

- Visual design for how this would look within the app market (maybe they're all different apps, but for apps that have an experimental channel, there would be two apps, e.g. "foobar" and "foobar experimental")

Then there's some code to write.

I don't think we plan on thinking hard about the above for a while, but if someone else takes a stab at it, we'd read/review. For now, this comment indicates that the issue is interesting but something the core team won't be addressing for the next couple of months, unless our priorities change, which I suppose they can if the whole community jumps up and down on this issue. :D

**Comment by mitar** [comment]
Do you use labels here?

**Comment by paulproteus** [comment]
We do use labels a little bit, but I don't know that we have a well-thought-out labeling scheme.

**Comment by neynah** [comment]
We don't yet but we should! :)

**Comment by ocdtrekkie** [comment]
A note from the mailing list of mine:

apps.sandstorm.io/?experimental=true on the App Store is a nice place to go for unapproved apps, a trend could be made to keep apps named "xxxxx Beta" unapproved, to separate them as true experimental apps from the main app market.

But of course, right now there'd be no way to switch a grain between those channels, because they'd be separate app IDs.
